### PR TITLE
59 feature check model for the report

### PR DESCRIPTION
**Pull request by Brian2397**
## Description

According to Prototype, was created a CHECK.
Was created a function that receives the title and a boolean value as parameters and inserts on the HTML document a check with the desired values.
If you get a Boolean parameter 'true', it shows a check.
If you get a boolean parameter 'false', it shows a no_check.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: app/scripts/contentscript.js [hunk 1]
```diff
@@ -83,6 +85,28 @@ function createCommitChart(){
...
 
+/**
+ * Create check model for the report element.
+ */
+function createCheckModel(text, boolCheck){
+    var content = document.getElementsByClassName("new-discussion-timeline experiment-repo-nav")
+    var repoContent = document.getElementsByClassName("repository-content")
+    var node = document.createElement('div')
+    var title = document.createElement('H2')
+    title.style = ('padding-left: 400px')
```
Comment (filipetoyoshima) [inline app/scripts/contentscript.js]: Why there is a `padding-left: 400px`?
The check model will be called in many places, not only on a 400px offset from left.

Please, center the title properly.

File: app/scripts/contentscript.js [hunk 2]
```diff
@@ -226,6 +250,7 @@ const init_with_no_request = () => {
 const hubcarePage = () => {
     cleanPageContent()
     createCommitChart()
+    createCheckModel('Have_License', true)
```
Comment (filipetoyoshima) [inline app/scripts/contentscript.js]: The prototype contains `"Have a License"` on the title, not `"Have_License"`.

File: app/scripts/contentscript.js (hunk 1 above)
Comment (filipetoyoshima) [reply app/scripts/contentscript.js]: I recommend the read: https://www.w3schools.com/css/css_align.asp

**Comment by filipetoyoshima** [comment]
Please, reopen PR when the corrections are made.

**Comment by FranciscoHeronildo** [comment]
Errors found and fixed.

**Pull request merged**